### PR TITLE
Pms v15 fixes

### PR DESCRIPTION
**Pull request by frederikgeth**
numerous fixes

### CODE REVIEW

**Comment by frederikgeth** [comment]
Some tests did not pass: 1085 passed, 83 failed, 14 errored, 0 broken.

File: src/form/apo.jl [hunk 1]
```diff
@@ -6,17 +6,17 @@ end
 
 
 "apo models ignore reactive power flows"
-function variable_mc_generation_on_off_reactive(pm::_PMs.AbstractActivePowerModel; kwargs...)
+function variable_mc_reactive_generation_on_off(pm::_PMs.AbstractActivePowerModel; kwargs...)
```
Comment (sanderclaeys) [inline src/form/apo.jl]: reactive at the end?

File: src/form/apo.jl [hunk 2]
```diff
@@ -277,14 +288,18 @@ function constraint_storage_loss(pm::_PMs.AbstractActivePowerModel, n::Int, i, b
 end
 
 function constraint_mc_storage_on_off(pm::_PMs.AbstractActivePowerModel, n::Int, i, pmin, pmax, qmin, qmax, charge_ub, discharge_ub)
+    @assert all(pmin.<=pmax)
```
Comment (sanderclaeys) [inline src/form/apo.jl]: Better to add this test at the template level and do it at once for all formulations?

File: src/form/apo.jl (hunk 1 above)
Comment (frederikgeth) [reply src/form/apo.jl]: the existing ones had this order?

File: src/form/apo.jl (hunk 2 above)
Comment (frederikgeth) [reply src/form/apo.jl]: remove

**Pull request merged**